lib/ui/painting/image_generator_registry.cc

```diff
@@ -33,6 +34,11 @@ ImageGeneratorRegistry::ImageGeneratorRegistry() : weak_factory_(this) {
       },
       0);
 #elif FML_OS_WIN
+  HRESULT status = CoInitializeEx(
```

Comment:
Why is this necessary?

Reply:
In a previous commit you had `shell/testing/tester_main.cc` initialize COM. Did that approach not work?